packages/wdio-rpc/package.json

```
@@ -0,0 +1,47 @@
+{
+  "name": "@wdio/rpc",
+  "version": "1.0.0",
```

Let's start with v9, I think Lerna will overwrite the version anyway but just to be on the same side.
```suggestion
  "version": "9.0.0",
```

---

done